Chesterton Fence: Add cartoon graphic by Oliver Widder

habere-et-dispertire opened the pull request:
Source: Oliver Widder, @owidder
Licence: CC-BY-3.0

habere-et-dispertire committed:
Cartoon graphic

Source: Oliver Widder, @owidder
Licence: CC-BY-3.0

ErikSchierboom reviewed (commented):
Thanks for the PR! I don't mind the image appearing somewhere in the document, but not at the top of the page and also not in this size.

habere-et-dispertire committed:
Resized geek-and-poke cartoon: Architecture

Author: Oliver Widder
Licence CC-BY-3.0: https://geek-and-poke.com/geekandpoke/2010/11/4/architecture.html

habere-et-dispertire committed:
Geek and poke - Architecture cartoon

Resized and minimized.

Original author: Oliver Widder
Licence CC_BY-30: https://geek-and-poke.com/geekandpoke/2010/11/4/architecture.html

habere-et-dispertire committed:
Moved and resized image

I could use some help as this is my first time trying to upload a file and reference it.

It seems images go in  docs/.imgs/. Is that right?

I couldn't get a relative web address to work. Is that right too?

I used <img src=

https://raw.githubusercontent.com/exercism/docs/79cdc57261c3c6209a16c55d3005fa7378d789e1/.imgs/architecture-min.png

Which feels like it may break.

habere-et-dispertire committed:
Update chestertons-fence.md

habere-et-dispertire committed:
Update chestertons-fence.md

habere-et-dispertire commented:
I'm new to the limitations of GitHub markdown. It appears you cannot float or style anything [if hosted on GitHub](https://stackoverflow.com/questions/48476312/float-an-image-next-to-a-table-in-github-flavored-markdown).

I am unsure how to place this where you'd like as I cannot seem to style or float it.
